pkg/cable/libreswan/libreswan.go

```diff
@@ -169,8 +170,11 @@ func (i *libreswan) Init() error {
 var trafficStatusRE = regexp.MustCompile(`.* "([^"]+)"[^,]*, .*inBytes=(\d+), outBytes=(\d+).*`)
 
 func retrieveActiveConnectionStats() (map[string]int, map[string]int, error) {
+	ctx, cancel := context.WithTimeout(context.TODO(), 5*time.Second)
```

Comment:
I thought `TODO()` was a better fit, one of its documented use-cases is “because the surrounding function has not yet been extended to accept a Context parameter” — ideally these functions should be extended to take a parent context as input.

Reply:
yeah perhaps.  `TODO` is fine.